pkg/controller/workload/workload_processor.go

```diff
@@ -486,6 +478,26 @@ func (p *Processor) storeServiceData(serviceName string) error {
 	sk.ServiceId = p.hashName.StrToNum(serviceName)
 	sv.LbPolicy = LbPolicyRandom
 	sv.EndpointCount = 0 // there are 0 endpoints in the initial state
+
+	if waypoint != nil {
+		sv.WaypointAddr = nets.ConvertIpByteToUint32(waypoint.GetAddress().Address)
+		sv.WaypointPort = nets.ConvertPortToBigEndian(waypoint.GetHboneMtlsPort())
```

Comment:
waypoint.GetHboneMtlsPort() -> KmeshWaypointPort

Reply:
here stored are ip and port of a waypoint service, not the waypoint backend port info.